erts/emulator/beam/erl_global_literals.h

```diff
@@ -24,8 +24,12 @@
 #define ERTS_LIT_OS_TYPE         0
 #define ERTS_LIT_OS_VERSION      1
 #define ERTS_LIT_DFLAGS_RECORD   2
+#define ERTS_LIT_EMPTY_TUPLE     3
```

Comment:
Calling `erts_get_global_literal(ERTS_LIT_EMPTY_TUPLE)` all the time seems a bit odd, perhaps we should make it `extern Eterm ERTS_LIT_XYZ` instead?

Reply:
Yes, that might be better. I can do that change.

Reply:
One disadvantage with referring to a global variable like ERTS_LIT_EMPTY_TUPLE in a lot of places for reading is that it may be harder to find where the variable is written to.

Reply:
I don't think that will be much of a problem in practice since there's so few of them and they're only written once. Searching for `erts_register_global_literal` will point you right to them.

I'll leave it up to @kjellwinblad though. :)

Reply:
I think I will keep it as it is since I agree with @sverker's comment and I don't think making it `extern Eterm ERTS_LIT_XYZ` will have any performance advantage in production builds (correct me if I'm wrong).

Reply:
`erts_get_global_literal` should be an inline function to not cost any performance. We don't have link-time-optimization enabled by default.

Reply:
I see, thanks. I have created a global variable for the empty tuple literal (1f5f661). The extern declaration is in `erl_global_literals.h` and the normal declaration is in `erl_init.c` as it contains many other global variables. Please let me know if you think there is a better place to place these.

Reply:
Good point! Fixed in 76843a8